### PR TITLE
first draft of impl report

### PR DESCRIPTION
**Pull request by svgeesus**
Captures a static snapshot of the automatic wpt tests, plus results of manual tests.

Summarizes current state of non-browser implementations.

### CODE REVIEW

**Comment by ProgramMax** [comment]
I'm still reviewing this. But I assume the ~$index.html file shouldn't be added.

**Comment by digitaltvguy** [comment]
I added a MediaInfp example and changed the MaxFALL #.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Chris Blume ***@***.***>
Sent: Wednesday, April 3, 2024 11:56:18 PM
To: w3c/PNG-spec ***@***.***>
Cc: Seeger, Chris (NBCUniversal) ***@***.***>; Review requested ***@***.***>
Subject: [EXTERNAL] Re: [w3c/PNG-spec] first draft of impl report (PR #445)


I'm still reviewing this. But I assume the ~$index.html file shouldn't be added.

File: Implementation_Report_3e/index.html [hunk 1]
```diff
@@ -0,0 +1,1046 @@
...
+                    </table>
+                
+                </section>
+
+                <section id="cicp">
+                    <h2>2. Coding Independent Code Points (CICP)</h2>
+                    <p>
+                        Originally developed for color space signalling in video workflows,
+                        CICP is also used in image formats such as HEIF, AVIF and JPEG-XL.
+                        CICP is more compact, and designed to signal use of specific mathematically represented
```
Comment (chrisn) [inline Implementation_Report_3e/index.html:521]:
```suggestion
                        CICP is more compact, and designed to signal the use of specific mathematically represented

```

File: Implementation_Report_3e/index.html [hunk 2]
```diff
@@ -0,0 +1,1046 @@
...
+                        color primaries, transfer functions, and matrix coefficients, and signal quantizations vs
```
Comment (chrisn) [inline Implementation_Report_3e/index.html:522]:
```suggestion
                        color primaries, transfer functions, matrix coefficients, and signal quantizations vs 

```

File: Implementation_Report_3e/index.html [hunk 3]
```diff
@@ -0,0 +1,1046 @@
...
+                </section>
+
+                <section id="cicp">
+                    <h2>2. Coding Independent Code Points (CICP)</h2>
+                    <p>
+                        Originally developed for color space signalling in video workflows,
+                        CICP is also used in image formats such as HEIF, AVIF and JPEG-XL.
+                        CICP is more compact, and designed to signal use of specific mathematically represented
+                        color primaries, transfer functions, and matrix coefficients, and signal quantizations vs 
+                        ICC profiles were originally created to matching color between different mediums (display, print, scan).
```
Comment (chrisn) [inline Implementation_Report_3e/index.html:523]:
```suggestion
                        ICC profiles, which were originally created to match color between different mediums (display, print, scan).

```

File: Implementation_Report_3e/index.html [hunk 4]
```diff
@@ -545,7 +545,7 @@ <h2>2. Coding Independent Code Points (CICP) from ITU-R H.273</h2>
                     </ul>
                     <p>Implemented in <a href="https://affinity.serif.com/en-us/photo/"><strong>Affinity Photo 2</strong></a></p>
                     <ul>
-                        <li><a href="https://affinity.help/photo2/en-US.lproj/index.html?page=pages/Appendix/fileformat.html&title=Supported%20file%20formats">Includes 32-bit HDR support (PNG specification - Third edition) for interchanging HDR broadcast imagery in a lossless format. Legacy PNG files exported from Photoshop with CICP data embedded in an ICC profile can be imported.</a></li>
+                        <li><a href="https://affinity.help/photo2/en-US.lproj/index.html?page=pages/Appendix/fileformat.html&title=Supported%20file%20formats">Includes 48-bit HDR support with an Alpha Channel (PNG specification - Third edition) for interchanging HDR broadcast imagery in a lossless format. Legacy PNG files exported from Photoshop with CICP data embedded in an ICC profile can be imported.</a></li>
```
Comment (svgeesus) [inline Implementation_Report_3e/index.html:548]: No, this is incorrect. Their own announcement (see screenshot in this implementation report) says **32 bits**. Per component!

File: Implementation_Report_3e/img/inspector-cLLi.png
Comment (svgeesus) [inline Implementation_Report_3e/img/inspector-cLLi.png:1]: 300 is indeed a better value, but I didn't have a test image containing such a value.

**Comment by digitaltvguy** [comment]
Ahhh. Very cool.  Thx


From: Chris Lilley ***@***.***>
Date: Friday, April 5, 2024 at 12:39 PM
To: w3c/PNG-spec ***@***.***>
Cc: Seeger, Chris (NBCUniversal) ***@***.***>, Review requested ***@***.***>
Subject: [EXTERNAL] Re: [w3c/PNG-spec] first draft of impl report (PR #445)

@svgeesus commented on this pull request.

________________________________

In Implementation_Report_3e/index.html<https://urldefense.com/v3/__https:/github.com/w3c/PNG-spec/pull/445*discussion_r1553983079__;Iw!!PIZeeW5wscynRQ!rjQrmFR13Fws4-p2HihGG356-KS5KNK1re7HkYisr_l4NVJQOYkEuXq696-eovI-2_b4LRjOqdd_7F-vrfjj2kmG5w$>:

> @@ -545,7 +545,7 @@ <h2>2. Coding Independent Code Points (CICP) from ITU-R H.273</h2>

                     </ul>

                     <p>Implemented in <a href="https://affinity.serif.com/en-us/photo/"><strong>Affinity<https://urldefense.com/v3/__https:/affinity.serif.com/en-us/photo/*22*3E*3Cstrong*3EAffinity__;JSUlJQ!!PIZeeW5wscynRQ!rjQrmFR13Fws4-p2HihGG356-KS5KNK1re7HkYisr_l4NVJQOYkEuXq696-eovI-2_b4LRjOqdd_7F-vrfhc46KC8Q$> Photo 2</strong></a></p>

                     <ul>

-                        <li><a href="https://affinity.help/photo2/en-US.lproj/index.html?page=pages/Appendix/fileformat.html&title=Supported%20file%20formats">Includes<https://urldefense.com/v3/__https:/affinity.help/photo2/en-US.lproj/index.html?page=pages*Appendix*fileformat.html&title=Supported*20file*20formats*22*3EIncludes__;Ly8lJSUl!!PIZeeW5wscynRQ!rjQrmFR13Fws4-p2HihGG356-KS5KNK1re7HkYisr_l4NVJQOYkEuXq696-eovI-2_b4LRjOqdd_7F-vrfhWux8A5w$> 32-bit HDR support (PNG specification - Third edition) for interchanging HDR broadcast imagery in a lossless format. Legacy PNG files exported from Photoshop with CICP data embedded in an ICC profile can be imported.</a></li>

+                        <li><a href="https://affinity.help/photo2/en-US.lproj/index.html?page=pages/Appendix/fileformat.html&title=Supported%20file%20formats">Includes<https://urldefense.com/v3/__https:/affinity.help/photo2/en-US.lproj/index.html?page=pages*Appendix*fileformat.html&title=Supported*20file*20formats*22*3EIncludes__;Ly8lJSUl!!PIZeeW5wscynRQ!rjQrmFR13Fws4-p2HihGG356-KS5KNK1re7HkYisr_l4NVJQOYkEuXq696-eovI-2_b4LRjOqdd_7F-vrfhWux8A5w$> 48-bit HDR support with an Alpha Channel (PNG specification - Third edition) for interchanging HDR broadcast imagery in a lossless format. Legacy PNG files exported from Photoshop with CICP data embedded in an ICC profile can be imported.</a></li>

No, this is incorrect. Their own announcement (see screenshot in this implementation report) says 32 bits. Per component!

**Review by ProgramMax — APPROVED** [review]
LGTM.
Also wow. I didn't realize implementation reports were so well designed.

File: Implementation_Report_3e/index.html [hunk 5]
```diff
@@ -0,0 +1,1059 @@
...
+                    for creating and editing APNG</a>.
+                </p>
+
+                <p>
+                    Implemented in <a href="https://www.xnview.com/en/xnviewmp/"><strong>XnSoft XnView MP</strong></a>
+                </p>
+
+                <!-- <p>
+                    Implemented in <a href="https://www.websitebuilderinsider.com/how-do-i-make-an-apng-in-photoshop/"><strong>Adobe Photoshop</strong></a> <em>although difficult to find: </em> in the Render Video window, select the “Photoshop Image Sequence” then select “PNG”. And then use APNGAsm.
+                </p> -->
```
[the remark this answers is not in the document]
Comment (svgeesus) [reply Implementation_Report_3e/index.html]: removed

File: Implementation_Report_3e/index.html [hunk 6]
```diff
@@ -0,0 +1,1046 @@
...
+        width:28em
+    }
+    footer {
+        margin-top: 1.5em;
+    }
+    #apng img.decorative {
+        float: right;
+    }
+    </style>
+    <!-- <link rel="stylesheet" href="http://www.w3.org/StyleSheets/TR/base"> -->
```
[the remark this answers is not in the document]
Comment (svgeesus) [reply Implementation_Report_3e/index.html]: removed

**Comment by digitaltvguy** [comment]
Hi All,

I will not be able to attend Monday’s meeting as I will be @ NAB.

Chris’s – Is there anything more you need from me on the implementation report?

Best,

Chris

**Pull request merged**